package.json

```diff
@@ -111,5 +111,6 @@
       "es5-ext@>=0.10.0 <0.10.63": ">=0.10.63",
       "follow-redirects@<=1.15.5": ">=1.15.6"
     }
-  }
+  },
+  "packageManager": "pnpm@9.3.0"
```

Comment:
Do you realise that this is still experimental?
https://nodejs.org/docs/latest-v18.x/api/all.html#all_packages_packagemanager

If I understand it correctly, this will be a very helpful feature!
On the other side we will loose the possibility to use pnpm as node version manager.

Reply:
@oliverschuerch did you test that pnpm does no longer work as a node version manager after this change?